src/runtime/virtcontainers/kata_agent.go

```diff
@@ -1531,6 +1532,21 @@ func (k *kataAgent) handleLocalStorage(mounts []specs.Mount, sandboxID string, r
 	}
 	return localStorages, nil
 }
+func (k *kataAgent) CheckandSetDmVerityDevice(m Mount) string {
+
+	for _, o := range m.Options {
+		if strings.HasPrefix(o, "extraoption=") {
+			virtVolume, err := types.ParseKataVirtualVolume(strings.TrimPrefix(o, "extraoption="))
+			if err != nil {
+				return ""
```

Comment:
Should we return error instead of null string here?

Reply:
Done.